spec/features/contact_us_submit_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'features/support/sign_up_helpers'
+require 'support/notifications_service'
+require 'support/send_help_email_use_case_spy'
+require'support/send_help_email_use_case'
+
+describe 'contact us page' do
+  include_examples 'notifications service'
+  include_examples 'send help email use case spy'
+  let(:user) { create(:user, :confirmed, :with_organisation) }
+
+  before do
+    sign_in_user user
+  end
+
+  it 'says request sent after pressing submit' do
```

Comment:
This looks like two assertions in one spec, its also saying `it 'sends a email to support'` to me.